src/customTheme.ts

```diff
@@ -99,6 +99,11 @@ const tertiaryTextDarkmode = coolGray['400']
 
 // ***** Collect constants, put them in a customTheme *****
 
+const themeConfig: ThemeConfig = {
+  initialColorMode: 'light',
+  useSystemColorMode: !!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE,
```

Comment:
If color mode is enabled with the feature flag, then the webapp will automatically be initialized with the color mode of your OS.

Reply:
I did see that the button changes the colors, but I guessed that was coming from the default chakra components, which supposedly all have their own darkmode styling.

Does this really matter? I intend to override all colors anyway, to make sure they're 100% according to the design... 😄

Reply:
That's Chakra's default for dark mode indeed. As we discussed before, better to override them manually first, and then we swipe Chakra's default tokens for dark mode by ours.